### ISSUE TITLE
Bug - Response is 200 for empty body

### ISSUE THREAD

**Issue by marcinziolo** [cdimascio/swagger-spring-functional#3]
If I send empty body.
In the following [code](https://github.com/cdimascio/openapi-spring-webflux-validator/blob/master/src/main/kotlin/io/github/cdimascio/swagger/Validate.kt#L101) `json` will be empty mono, `flatMap` won't be invoked at all, therefore validation and handler won't be invoked too. Endpoint returns empty body with 200 http status code.

```
        fun validate(handler: (T) -> Mono<ServerResponse>): Mono<ServerResponse> {
            val success = { json: String -> objectMapperFactory().readValue(json, bodyType) }
            val json = request.body(BodyExtractors.toMono(String::class.java))
            return json.flatMap { validator.validate(request, it) ?: handler(success(it)) }
        }
```

**Comment by marcinziolo** [comment]
Here is [PR](https://github.com/cdimascio/openapi-spring-webflux-validator/pull/4) with the fix. Could you take a look on it?

**Comment by cdimascio** [comment]
thanks for the fix! its live in v2.1.1

**Comment by marcinziolo** [comment]
Thanks for the super quick release.
